app/assets/javascripts/jquery.multi-select.js

```diff
@@ -75,13 +75,15 @@ function sanitize(value){
 $(document).on('click', '.ms-select-all', function () {
   // can't use multiSelect('select_all') because it adds filtered out items too.
     $(this).closest('.form-group').find('.ms-selectable .ms-list :visible').click();
+    $(this).tooltip('hide');
```

Comment:
Now after clicking and another refocusing, it stays open.
Should we destroy as well? I think the tooltips get created on the fly now, so it will get recreated right?

Reply:
hmmm, on additional testing looks like not all cases are properly handled:\ i'll work on it a bit further

Reply:
Turns out you can't destroy delegated tooltips - https://getbootstrap.com/docs/3.4/javascript/#tooltip-destroy. updated to hide on removal as well.